tests/test_chunking_embedding.py

```
@@ -0,0 +1,75 @@
+        chunk_text("abc", max_length=0, overlap=0)
+    with pytest.raises(ValueError, match="max_length must be greater than 0"):
+        chunk_text("abc", max_length=-1, overlap=0)
+    # overlap < 0
+    with pytest.raises(ValueError, match="overlap must be >= 0"):
+        chunk_text("abc", max_length=5, overlap=-1)
+    # overlap >= max_length
+    with pytest.raises(ValueError, match="overlap must be less than max_length"):
+        chunk_text("abc", max_length=5, overlap=5)
+    with pytest.raises(ValueError, match="overlap must be less than max_length"):
+        chunk_text("abc", max_length=5, overlap=6)
+import pytest
```

Ruff (0.12.2)

Redefinition of unused `pytest` from line 3

Remove definition: `pytest`

(F811)